tests/pathfinding/test_blockchain_integration.py

```diff
@@ -74,17 +81,27 @@ def test_pfs_with_mocked_client(
         _settle_timeout,
     ) in channel_descriptions_case_1:
         # order is important here because we check order later
-        channel_identifier = clients[p1_index].open_channel(clients[p2_index].address)
-        channel_identifiers.append(channel_identifier)
+        channel_id = create_channel(clients[p1_index], clients[p2_index])[0]
+        channel_identifiers.append(channel_id)
 
-        clients[p1_index].deposit_to_channel(clients[p2_index].address, p1_deposit)
-        clients[p2_index].deposit_to_channel(clients[p1_index].address, p2_deposit)
+        for address, partner_address, amount in [
+            (clients[p1_index], clients[p2_index], p1_deposit),
+            (clients[p2_index], clients[p1_index], p2_deposit),
+        ]:
+            custom_token.functions.approve(token_network.address, amount).transact(
+                {"from": address}
+            )
+            return token_network.functions.setTotalDeposit(
```

Comment:
This `return` is the reason for the code coverage decrease. And it really should not be there, of course! Thanks for spotting the coverage decrease!